priv/posts/marketing/20210331201645_getting-to-know-us-prodriguez.md

```diff
@@ -11,7 +11,7 @@ Getting to know Pehuén Rodriguez
 ---
 
 
-<img src="/images/getting-to-know/prodriguez.png" class="img-fluid" alt="Pehuén Rodriguez"/>
+<img src="/images/getting-to-know/prodriguez.png" width="100%" height="auto" alt="Pehuén Rodriguez"/>
```

Comment:
Why these changes?

Reply:
Same as above, Markdown sanitizer removes the class attribute

Reply:
Oh... right, you mentioned that and I forgot it... excellent attention to detail on your part!